src/components/DayPickerSingleDateController.jsx

```diff
@@ -508,6 +532,17 @@ export default class DayPickerSingleDateController extends React.PureComponent {
     return { currentMonth, visibleDays };
   }
 
+  shouldDisableMonthNavigation(date, visibleMonth) {
+    if (!date) return false;
```

Comment:
Although it makes it more obvious, it might not be needed this early return, as the `isDayVisible` already does that when `date` is not a moment object